### PR TITLE
Feat: Card and hero image snippet were added

### PR DESCRIPTION
**Pull request by DanielGovrin**

### CODE REVIEW

**Comment by alisey** [comment]
What's the context for this? It doesn't follow the guidelines we've set for snippets. The boards that served as examples `hero-center.board.tsx` and `hero-flex.board.tsx` deleted with no explanation. `services-section.board.tsx` is deleted, but `services-section.board.css` is left.

**Comment by DanielGovrin** [comment]
Apologies, I misunderstood something that Dana told me. I’ll add those files back to the repo.

**Comment by Dana-Levy** [comment]
hero-center.board.tsx
hero-flex.board.tsx 

These aren't the final snippets

File: _codux/boards/services-section.board.tsx [hunk 1]
```diff
@@ -76,4 +76,4 @@ export default createBoard({
         windowWidth: 1150,
         windowHeight: 650,
     },
-});
+});
```
Comment (0xRezaa) [inline _codux/boards/services-section.board.tsx]: please add the end of line break

**Comment by 0xRezaa** [comment]
Just a recording of me reviewing the hero-image visually

https://github.com/user-attachments/assets/c9a97264-6482-4a72-8f34-c24df6d625ae

File: _codux/boards/card-collection-1.board.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            <h1 className="card-collection-title">
+                Write a Title Here.
+                <br /> Click to Edit and Add Your Own Text.
+            </h1>
+            <div className="card-collection-items">
+                <div className="card">
+                    <div className="card-image-box">
+                        <img
+                            src="https://static.wixstatic.com/media/8586f3_f6d2538ac6374f71b6dc29511c5cd1a3~mv2.png/v1/fill/w_860,h_600/Frame%206.png"
+                            alt="Delicious Margherita Pizza"
```
Comment (0xRezaa) [inline _codux/boards/card-collection-1.board.tsx]: I think that having an alt value is a benefit as we insentivise the users to set them, maybe we want to use a different value than this.

**Comment by 0xRezaa** [comment]
Just a recording of me reviewing the hero-title visually

https://github.com/user-attachments/assets/d3d972b1-b499-4331-92d1-7401c6914b69

**Comment by 0xRezaa** [comment]
Just a recording of me reviewing the card-collection-1 visually


https://github.com/user-attachments/assets/bd3ebd54-987c-48ca-90ab-0e40aa7955ed

**Comment by 0xRezaa** [comment]
Just a recording of me reviewing the card-collection-2 visually

https://github.com/user-attachments/assets/5e67cac4-09aa-448e-aaa8-ea29ac0e7f4f

File: _codux/boards/card-collection-2.board.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
+.card-collection {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    font-family: Arial, sans-serif;
+    min-height: 800px;
+    max-height: 700px;
+
+    @media (width<= 1024px) {
```
Comment (0xRezaa) [inline _codux/boards/card-collection-2.board.css]: Is this supposed to be `width < 1024px`

See the recording

https://github.com/user-attachments/assets/8115cb32-20da-415a-bfcf-a67081bc34c9

**Review by 0xRezaa — APPROVED** [review]

**Pull request merged**